src/Contrib/Grpc/GrpcTransport.php

```diff
@@ -38,15 +38,18 @@ final class GrpcTransport implements TransportInterface
     private readonly array $metadata;
     private readonly Channel $channel;
     private bool $closed = false;
+    private Timeval $exportTimeout;
 
     public function __construct(
         string $endpoint,
         array $opts,
         private readonly string $method,
         array $headers = [],
+        int $timeoutMillis = 500,
     ) {
         $this->channel = new Channel($endpoint, $opts);
         $this->metadata = $this->formatMetadata(array_change_key_case($headers));
+        $this->exportTimeout = new Timeval($timeoutMillis * 1000);
```

Comment:
Can we use a const here, to make it clear we're converting from milliseconds to microseconds? We don't have a const for this yet in `API\Common\Time\ClockInterface`, but I think that's the place to put it.